dts/arm/nxp/nxp_lpc54xxx.dtsi

```diff
@@ -45,9 +53,9 @@
 			reg = <0x20020000 0x8000>;
 		};
 
-		sramx:memory@40000000{
+		sramx:memory@4000000{
 			compatible = "mmio-sram";
-			reg = <0x40000000 0x8000>;
+			reg = <0x4000000 0x8000>;
```

Comment:
Stray change?

Reply:
The value was incorrect. I have fixed the address